JULIERAJ/KIDS-FIRST-Front
# Feature/71/refactor sidebar component

# PR Template Frontend

## One Line Description
Refactored Sidebar to use react-router paths. Updated sidebar icons from google fonts to svg.

## Requirements
Fix routing for sidebar links and bug with loading sidebar icons.

## Notes
Icons in side bar load faster now and do not appear blank while loading. However, when switching colors between active and not active state I can see a split-second delay.

## Test Steps
- Log in to view Dashboard and observe icons in sidebar. There should not be any delay in loading.
- Click on icons to view style changes.

## Recoding:
https://github.com/user-attachments/assets/0ef1adf4-e0b7-4c49-9416-4f99e9651a8a
